core/templates/dev/head/components/top_navigation_bar/top_navigation_bar_directive.html

```diff
@@ -25,26 +25,27 @@
       <li ng-show="navElementsVisibilityStatus.I18N_TOPNAV_LIBRARY" class="oppia-clickable-navbar-element">
         <a class="oppia-navbar-tab oppia-navbar-tab-content" href="/library" translate="I18N_TOPNAV_LIBRARY"></a>
       </li>
-      <li uib-dropdown ng-show="navElementsVisibilityStatus.I18N_TOPNAV_ABOUT" class="oppia-navbar-clickable-dropdown protractor-test-about-oppia-list-item">
-        <a class="oppia-navbar-tab"
+      <li ng-class="{'open' : activeMenu === 'aboutMenu'}" uib-dropdown ng-show="navElementsVisibilityStatus.I18N_TOPNAV_ABOUT" class="oppia-navbar-clickable-dropdown protractor-test-about-oppia-list-item">
+        <a ng-keydown="handleNavigation($event,'aboutMenu','shiftTab','close');handleNavigation($event,'aboutMenu','enter','open')" class="oppia-navbar-tab"
```

Comment:
Refactor and reformat as follows:

    ng-keydown="onMenuKeypress($event, 'aboutMenu', {shiftTab: 'close', enter: 'open'})"

Note the space after each comma.

Ditto below.

Reply:
Done. @seanlip PTAL.

Thanks.